### PR TITLE
feat: activate semversionning on windowsservercore-1809 image

### PR DESCRIPTION
**Pull request by lemeurherve**

### CODE REVIEW

**Comment by dduportal** [comment]
Interesting: seems like that https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildDockerAndPublishImage.groovy#L24 is false in this case.

**Comment by lemeurherve** [comment]
Make sense in retrospect.

File: Jenkinsfile_k8s [hunk 1]
```diff
@@ -1,57 +1,12 @@
 parallel(
```
Comment (lemeurherve) [inline Jenkinsfile_k8s:1]:
```suggestion
@Library('pipeline-library@pull/390/head') _

parallel(
```
Testing https://github.com/jenkins-infra/pipeline-library/pull/390

**Comment by lemeurherve** [comment]
Added next semver version as log only in PRs: https://github.com/jenkins-infra/pipeline-library/pull/390

**Review by dduportal — APPROVED** [review]

**Comment by lemeurherve** [comment]
Main build failing as the credential.helper isn't correctly configured for Windows [in the pipeline lib](https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildDockerAndPublishImage.groovy#L118). (empty env vars, certainly a bad interpolation)

**Pull request merged**